orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -1168,7 +1168,7 @@ Feature: User can specify that a value is equalTo a required value
   Scenario: 'equalTo' with a non-contradicting 'aValid' constraint should be successful
```

Comment:
Unlike other features scenario titles in this feature still feature use of aValid

Reply:
Also this scenario just says 'aValid' constraint, not specifying what type of fincancial code